### ISSUE TITLE
Add trim box

### ISSUE THREAD

**Issue by mxyolande** [avito-tech/Paparazzo#5]
i saw the PhotoTweak, but the i add the rotate 90 fail, and i enjoy the style of Paparazzo
so, is there any plan to add the trim box.
why Apple do not open the permission of Photo.app editing model 😂

**Comment by vkaltyrin** [comment]
Hi @mxyolande, thanks for your suggestion. Could please specify what do you mean by a "trim box"?
What features would you like us to implement?

**Comment by mxyolande** [comment]
yeah, you are right. Crop images by dragging the edges of a grid overlay. 😁

**Comment by HiveHicks** [comment]
We will consider adding this this feature in the future.
